src/gluonts/mx/model/seq2seq/_mq_dnn_estimator.py

```diff
@@ -135,6 +140,9 @@ class MQCNNEstimator(ForkingSeq2SeqEstimator):
         Determines whether to use IQF or QF. (default: True).
     batch_size
         The size of the batches to be used training and prediction.
+    train_sampler
+        Controls the sampling of windows during training.
```

Comment:
Maybe it makes sense to similarly expose the `validation_sampler` that the upper class exposes